crates/bevy_ui/src/widget/image.rs

```diff
@@ -119,6 +119,55 @@ impl ImageNode {
         self
     }
 
+    /// `with_rect` crops an `ImageNode` to the portion described by
+    /// a `Rect`. This can be applied to `ImageNode`s created from
```

Comment:
```suggestion
    /// the provided `Rect`, measured from the top-left corner. This can be applied to `ImageNode`s created from
```
Not sure about the top-left bit; please fill in the correct answer bit of information there if it's wrong!